skema/program_analysis/CAST/matlab/tests/test_matlab_parse_tree.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import os
+from pathlib import Path
+
+from tree_sitter import Language, Parser
+
+# test parser
+from skema.program_analysis.CAST.matlab.matlab_to_cast import MATLAB2CAST
+
+# test for existence of shared grammar object
+from skema.program_analysis.tree_sitter_parsers.build_parsers import INSTALLED_LANGUAGES_FILEPATH
+if os.path.exists(INSTALLED_LANGUAGES_FILEPATH):
+    print('Shared object at: ' + str(INSTALLED_LANGUAGES_FILEPATH))
+else:  
+    # If not found, create it
+    print('Did not find shared object at ' + str(INSTALLED_LANGUAGES_FILEPATH))
+    from skema.program_analysis.tree_sitter_parsers.build_parsers import build_parsers
+    build_parsers('--matlab')
```

Comment:
Ignore if already working, but this may need to `build_parsers(["matlab"])`

Reply:
Implemented.   Good call.